test/images/candlepin

```diff
@@ -1 +1 @@
-candlepin-392368746586c79a7bb5aa6fa38de0b18121d51a.qcow2
\ No newline at end of file
+candlepin-29a74a0221a854a68e1b8e761f9e74c6a30654c0.qcow2
```

Comment:
Testing doesn't use the images on these branches, only from master. Did these changes slip in here by accident or is there a reason to change the images on the rhel-7-4 branch?